### PR TITLE
Add spelling corrections for Policy and variants.

### PR DESCRIPTION
**Pull request by cfi-gb**
See e.g.:
- https://github.com/search?q=polciy&type=code
- https://grep.app/search?q=polciy

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -28428,6 +28428,7 @@ pojrectors->projectors
 pojrects->projects
 poket->pocket
 polariy->polarity
+polciy->policy
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28431]: I would suggest adding
```
polcy->policy
polcies->policies
```

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:28431]: Thanks a lot for the additional suggestions. These would extend the scope of this PR which items i haven't found / noticed on my own. If this is o.K. from your / project side i'm happy to apply them.

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:28431]: It's certainly OK from my side. Your choice.

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:28432]: Thanks a lot, updated the PR accordingly. :+1:

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @cfi-gb.

**Pull request merged**